source/includes/gcp/_k8_daemonsets.md

```diff
@@ -0,0 +1,139 @@
+### Daemon Sets
+
+<!-------------------- LIST DAEMON SETS -------------------->
+
+#### List daemon sets
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/k8s/anenvironment/daemonsets"
```

Comment:
`anenvironment` ---> `an_environment`: I guess without the underscore at first site its not evident its an env

Reply:
a4d0afc081c7fa8eacad5f473c9865970cb197d8